lib/DDGC/DB/Result/InstantAnswer.pm

```diff
@@ -347,6 +347,12 @@ column src_id => {
     for_endpt => 1
 };
 
+column is_stackexchange => {
```

Comment:
Because it's an independent field, so we treat it as independent column. At least I think it is independent, from my understanding :)